### DISCUSSION TITLE
Annoying Steam icons

### DISCUSSION THREAD

**Discussion by iskandar9ll** [Whisky-App/Whisky#581] [Q&A] [answered]
is there anyway to get rid of those steam icons on menu bar. Holding Command and draging them out didnt help.
<img width="644" alt="Screenshot 2023-11-01 at 09 48 40" src="https://github.com/Whisky-App/Whisky/assets/148012608/7737e22a-ab09-4430-b7a1-fceaebe255db">

**Comment by mavethee** [comment, marked as answer]
Usually there's only one at best.
Whisky -> `File` -> `Kill All Bottles`
Re-run Whisky, open Steam _just once_.

If you want to get rid of any icon wine related to show up, you can use HiddenBar app for example:
![CleanShot 2023-11-01 at 09 58 26@2x](https://github.com/Whisky-App/Whisky/assets/47083527/01dcebdb-7481-4d76-8c18-f5d7586a0238)
![CleanShot 2023-11-01 at 09 58 08@2x](https://github.com/Whisky-App/Whisky/assets/47083527/c28a0bcd-a272-4cad-bc99-b5aef4c86ec9)

**Reply by iskandar9ll** [reply]
THank you, hiddenbar app helped. Couldnt manage with your instructions on kill all bottles tho

**Reply by josepheiba** [reply]
I hope that someday we get rid of the menu bar icon, I think that Steam can live on the dock just like all other apps, just for aesthetic's sake, it's not a problem now, cause I only open it when I'm gaming and I play full screen, and Whisky is nice enough to not add the app and games icons to my dock, and that keeps things organized

**Reply by IsaacMarovitz** [reply]
Menu bar icons are Wine's doing not Whisky's. We could potentially patch that out of our build of Wine.

**Reply by josepheiba** [reply]
that would be amazing, but I guess you have to check with other users, maybe they like it as it is. also if you close steam by the [x] button, it can't be opened from the dock, so it may require some additional work.
